src/mlang/backend_ir/bir_interpreter.ml

```diff
@@ -479,10 +479,11 @@ module Make (N : Bir_number.NumberInterface) = struct
                 | TableVar _ -> assert false
                 (* should not happen *)
               with Not_found ->
```

Comment:
This was needed when variables where duplicated because some where refering to the assigned variable in M code instead of the declaration. It happens in particular with variable defined in corrective rules.
But since variable unification this should not be a problem anymore, I reverted this change.